test/onnx/test_fx_dynamic_with_onnxruntime.py

```diff
@@ -149,6 +151,8 @@ def tearDown(self):
         " typing.Sequence[int], torch.Tensor], as [None, None]:"
     )
     def test_shufflenet_v2_dynamic_axes(self):
+        import torchvision
```

Comment:
I think we always have torchvision?

Reply:
I didn't locally, and this test is disabled.

Reply:
This test is skipped for other reasons, but I will leave a comment here.